### PR TITLE
Adds a fix for long loadingtimes upon connecting

### PR DESCRIPTION
**Pull request by StripesOO7**
Adds a fix for long loadingtimes upon connecting and touching MANY items at once resulting in a timeout and thus infinite loop. (fix by Mysteryem)

### CODE REVIEW

File: scripts/autotracking/archipelago.lua [hunk 1]
```diff
@@ -11,6 +11,32 @@ SLOT_DATA = nil
 LOCAL_ITEMS = {}
 GLOBAL_ITEMS = {}
 
+function onClearHandler(slot_data)
+    clear_timer = os.clock()
```
Comment (black-sliver) [inline scripts/autotracking/archipelago.lua]: I have not tested the suggestion, but if you don't put `local clear_timer` *anywhere*, it's a global, which is bad.
In my testing i made `clear_timer` local to the module, but local to the function would be even better.

```suggestion
    local clear_timer = os.clock()
```

Comment (StripesOO7) [reply scripts/autotracking/archipelago.lua]: Sound good. I didn't pay attention to the namespace behavior.

File: scripts/autotracking/archipelago.lua [hunk 2]
```diff
@@ -11,6 +11,32 @@ SLOT_DATA = nil
...
+    local ok, err = pcall(onClear, slot_data)
+    -- Enable tracker updates again.
+    if ok then
+        -- Defer re-enabling tracker updates until the next frame, which doesn't happen until all received items/cleared
+        -- locations from AP have been processed.
+        local handlerName = "AP onClearHandler"
+        local function frameCallback()
+            ScriptHost:RemoveOnFrameHandler(handlerName)
+            Tracker.BulkUpdate = false
+            forceUpdate()
```
Comment (Cyb3RGER) [inline scripts/autotracking/archipelago.lua:28]: This function isn't implemented

**Comment by Cyb3RGER** [comment]
I'll close this PR for now because PopTracker has improved performance and added the `Tracker.AllowDeferredLogicUpdate` feature. I'd prefer to use this in the template pack rather than relying on a more complicated workaround.

**Comment by black-sliver** [comment]
@StripesOO7 you could instead add it as code snippet to the PopTracker documentation. I feel like the documentation on this (behaviour) is not very detailed at the moment.

That being said I hope to extend the new flag `Tracker.AllowDeferredLogicUpdate` to work outside of bulk update, which would fix the majority of the slowdown related to AP item/location sync (and spam) as well without having to use bulk update mode.

**Pull request closed without merging**